starter-code/server.js

```diff
@@ -34,15 +38,15 @@ app.use(express.static('./public'));
 // REVIEW: Routes for requesting HTML resources
 app.get('/new', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // response is 5,sendFile is 5 because it's the response. The method corresponding in article.js is the loadAll()method on line 35. Read is what is corresponding with this line.
```

Comment:
diagram 2 request & 5 response
loadAll() is not being called here, there isn't a corresponding article.js method